src/Tests/dotnet.Tests/WindowsInstallerTests.cs

```diff
@@ -131,58 +131,32 @@ public void InstallMessageDispatcherProcessesMessages()
+            Assert.Equal(expectedStatus, status);
 
-            if (!shouldBeSigned)
+            if (expectedStatus != 0)
             {
-                Assert.Equal(expectedError, new Win32Exception(Marshal.GetLastWin32Error()).Message);
+                Assert.Equal(expectedError, Marshal.GetPInvokeErrorMessage(status));
             }
         }
 
         [WindowsOnlyTheory]
-        [InlineData("dotnet_realsigned.exe")]
-        [InlineData("dotnet_fakesigned.exe")]
-        public void IsSignedByTrustedOrganizationOnlyVerifiesTheSubjectOrganization(string file)
-        {
-            Assert.True(AuthentiCode.IsSignedByTrustedOrganization(Path.Combine(s_testDataPath, file), AuthentiCode.TrustedOrganizations));
-        }
-
-        [WindowsOnlyFact]
-        public void IsSignedBytTrustedOrganizationVerifiesNestedSignatures()
+        [InlineData("dotnet_realsigned.exe", true)]
+        // The file is dual signed, but the cert APIs only look at the first certificate chain so it will pick up the
+        // .NET Foundation certificate that terminates in a DigiCert root certificate and return false.
+        [InlineData("dual_signed.dll", false)]
+        public void ItVerifiesThatTheRootCertificateIsTrusted(string file, bool expectedResult)
         {
-            Assert.True(AuthentiCode.IsSignedByTrustedOrganization(Path.Combine(s_testDataPath, "dual_signed.dll"),
-                "Foo", "Bar", "Microsoft Corporation")); ;
-            Assert.True(AuthentiCode.IsSignedByTrustedOrganization(Path.Combine(s_testDataPath, "dual_signed.dll"),
-                "Foo", "Bar", "WiX Toolset (.NET Foundation)"));
-        }
-
-        [WindowsOnlyFact]
-        public void GetCertificatesRetrievesNestedSignatures()
-        {
-            var certificates = AuthentiCode.GetCertificates(Path.Combine(s_testDataPath, "triple_signed.dll")).ToArray();
-
-            Assert.Equal("CN=Microsoft Corporation, OU=MOPR, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", certificates[0].Subject);
-            Assert.Equal("sha1RSA", certificates[0].SignatureAlgorithm.FriendlyName);
-            Assert.Equal("CN=Microsoft 3rd Party Application Component, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", certificates[1].Subject);
-            Assert.Equal("sha256RSA", certificates[1].SignatureAlgorithm.FriendlyName);
-            Assert.Equal("CN=Microsoft Corporation, OU=MOPR, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", certificates[2].Subject);
-            Assert.Equal("sha256RSA", certificates[2].SignatureAlgorithm.FriendlyName);
-        }
-
-        [WindowsOnlyFact]
-        public void GetCertificatesRetrievesNothingForUnsignedFiles()
-        {
-            var certificates = AuthentiCode.GetCertificates(Assembly.GetExecutingAssembly().Location);
-
-            Assert.Empty(certificates);
+            X509Certificate certificate = X509Certificate.CreateFromSignedFile(Path.Combine(s_testDataPath, file));
```

Comment:
Wrap with `using` statement.

Reply:
Most X509 pieces are removed in the new iteration, but I did add usings for the few that remain